clients/drcachesim/tracer/raw2trace.h

```diff
@@ -933,6 +1037,10 @@ class raw2trace_t {
 
         // Used to delay a thread-buffer-final branch to keep it next to its target.
         std::vector<trace_entry_t> delayed_branch;
+        // This is the first step of optimization of using delayed_branch vector
+        // Checking the bool value is cheeper than delayed_branch.empty()
+        // In general it's better to avoid this vector at all.
+        bool delayed_branch_empty_ { true };
```

Comment:
Best to match the existing code using ` = ` rather than list init.

Reply:
done  #6176